ext/puma_http11/extconf.rb

```diff
@@ -12,4 +12,10 @@
   end
 end
 
+BROKEN_VERSIONS = %w(2.2.7 2.3.4 2.4.1)
```

Comment:
No-no, the 2.2.0-2.2.6, 2.3.0-2.3.3 and 2.4.0 are good and unaffected by the bug, I've checked that pretty carefully.